features/paint.yml

```
@@ -0,0 +1,9 @@
+name: paint()
+description: The `paint()` CSS function paints an element's background or border using a custom paint worklet.
```

I didn't really know what "paint" meant here, so after reading the docs…

```suggestion
description: The `paint()` CSS function creates a custom image, drawn using a paint worklet, as in an element's `background-image` or `border-image` property.
```

I'm not sure that's a good description, but I think it adds a little more info. What do you think?

---

Thanks, feels better. I'm a bit confused by the "as in" part at the end. I'll think of it and push a new commit.